front50-gcs/src/main/java/com/netflix/spinnaker/front50/config/GcsConfig.java

```diff
@@ -129,49 +108,16 @@ public RestTemplate restTemplate() {
     return new RestTemplate();
   }
 
-  @Bean
-  public ApplicationDAO applicationDAO(GcsStorageService service, Registry registry) {
-    return new DefaultApplicationDAO(service, Schedulers.from(Executors.newFixedThreadPool(20)), APPLICATION_REFRESH_MS, registry);
-  }
-
-  @Bean
-  public ApplicationPermissionDAO applicationPermissionDAO(GcsProperties gcsProperties, Registry registry) {
+  @Override
+  public ApplicationPermissionDAO applicationPermissionDAO(StorageService storageService,
+                                                           StorageServiceConfigurationProperties storageServiceConfigurationProperties,
+                                                           Registry registry) {
     GcsStorageService service = googleCloudStorageService(ApplicationPermissionDAO.DEFAULT_DATA_FILENAME, gcsProperties);
```

Comment:
It looked like GCS has deviated from the defaults for the `ApplicationPermissionDAO` hence the override.